src/2048_Expert/2048-expert.css

```diff
@@ -0,0 +1,80 @@
+/*
+Author: Zihan Zhong
+version: 0
+ */
+
+ *{
+    margin: 0;
+    padding: 0;
+ }
+
+ html{
+    user-select:none;
+ }
+
+
+ /* colour scheme follow the convention of 2048 games */
+ body{
```

Comment:
Nit: please use a space before the curly brackets in the CSS to ensure consistency with the rest of the codebase